src/mumble/Settings.h

```diff
@@ -431,10 +431,15 @@ struct Settings {
 	QByteArray qbaPTTButtonWindowGeometry = {};
 
 	// Network settings
-	bool bTCPCompat   = false;
-	bool bReconnect   = true;
-	bool bAutoConnect = false;
-	bool bQoS         = true;
+	bool bTCPCompat                   = false;
+	bool bUdpVoiceStreamInboundOnly   = false;
+	bool bUdpVoiceStreamOutboundOnly  = false;
+	bool bUdpVoiceStreamBidirectional = false;
+	bool bUdpVoiceStreamAuto          = true;
```

Comment:
I don't think we need "voice" as part of the names. This affects the UDP stream which happens to be what voice is sent through (normally). But I don't see a need to include that in the variable names.

Reply:
Remove 'Voice' in all places.